pkgs/desktops/xfce4-14/default.nix

```diff
@@ -110,4 +108,10 @@ makeScope newScope (self: with self; {
   xfce4-namebar-plugin = callPackage ../xfce/panel-plugins/xfce4-namebar-plugin.nix { };
 
   xfce4-windowck-plugin = callPackage ../xfce/panel-plugins/xfce4-windowck-plugin.nix { };
+
+
+  ## ALIASES
+
+  # added 2019-08-18
+  xfce4-mixer = throw "deprecated 2019-08-18: obsoleted by xfce4-pulseaudio-plugin";
```

Comment:
Will this not break `nix-env`?

Reply:
I think the panel plugins are loaded from `/run/current-system/sw` so `nix-env` wouldn't have been supported.

Reply:
Hmm, I misremembered. The issue was with `trace`: https://github.com/NixOS/nixpkgs/pull/19315